### ISSUE TITLE
Mnemonic Phrase mismatch between display and confirmation

### ISSUE THREAD

**Issue by xendarboh** [aurowallet/auro-wallet-browser-extension#24]
In chrome browser (Linux) with fresh install of Auro Wallet extension, during setup there can be a mismatch of phrase words between the "Backup Mnemonic Phrase" display and the following confirmation step requiring input to proceed.

This occurs after waiting on the first display screen for a certain amount of time, for example the time it takes to manually write down the words. (A separate issue, is that floating windows of the wallet will disappear upon mouse re-focus, so can't use another application on the same computer to record the phrase.)

This does not occur if more quickly proceeding from backup to confirmation screens. Not sure if it will occur every time, but with waiting between steps, I could reproduce the issue multiple times.

For example, this "display" step:
![image](https://github.com/aurowallet/auro-wallet-browser-extension/assets/1435589/1dd5d145-7f25-4f12-98c5-54e9dd45239e)

and this "confirmation" step:
![image](https://github.com/aurowallet/auro-wallet-browser-extension/assets/1435589/15eed344-e67e-452b-8605-e2902b30f3ea)

**Comment by wjdfx** [comment]
Thanks for your feedback. We aware It's a bug cause Manifest V3. Will fix this at next version.

**Comment by SvenMeyer** [comment]
@wjdfx This is a SEVERE problem ! What if somebody saves the wrong seed words, has to restore, and then it does not work ! He would loose all funds !
Ideally there would be a warning on the affected version(s) and even shared (at lest) on twitter and discord !

**Comment by xendarboh** [comment]
Thanks, I look forward to the next version. Happy to test.
It's possible that this issue and #25 are expressions of the same bug, a 30s expiration.
The wallet is unusable as it is and I'm looking at downgrading the extension until resolved.
